src/bindings/c/docs/OpenVINO_Runtime_C_API_User_Guide.md

```diff
@@ -0,0 +1,62 @@
+# User Guide of OpenVINO C API
+
+> **NOTE**: This guide provides a simplified instroduction about OpenVINO C API.
+C API provides simplified interface for OpenVINO functionality that allows to:
+
+- handle the input network models & input/output tensors.
+- load and configure OpenVINO plugins based on device name.
+- perform inference in synchronous and asynchronous modes with arbitrary number of infer requests (the number of infer requests may be limited by target device capabilities).
+
+## Supported OSes
+
+Currently the OpenVINO C API is supported on Ubuntu* 20.04, Microsoft Windows* 10 and CentOS* 7.3 OSes.
+Supported Python* versions:
+
+- On Ubuntu 20.04: 2.7, 3.5, 3.6
+- On Windows 10: 3.5, 3.6
+- On CentOS 7.3: 3.4, 3.5, 3.6
```

Comment:
We don't have special requirements for C API, which are different from OpenVINO requirements. We don't need to have it.